nflow-engine/src/main/java/io/nflow/engine/config/db/SqlServerDatabaseConfiguration.java

```diff
@@ -97,8 +97,8 @@ public boolean hasUpdateableCTE() {
     }
 
     @Override
-    public String forUpdateInnerSelect() {
-      return "";
+    public String forUpdateInnerSelect(boolean skipLocked) {
+        return skipLocked ? " with (updlock,readpast)" : " with (updlock)";
```

Comment:
This is always called with `true`. Is this conditional needed?

Reply:
I think it is more clear this way. But I could rename the method to forUpdateInnerSelectSkipLocked and remove the boolean. Not sure if we ever need the false option

Reply:
IMO remove boolean and rename if needed